hack/update-codegen.sh

```diff
@@ -19,6 +19,12 @@ set -o nounset
 set -o pipefail
 
 export GO111MODULE=on
+# If we run with -mod=vendor here, then generate-groups.sh looks for vendor files in the wrong place.
```

Comment:
Just out of curiosity, can you explain a bit more on this comment? Where would the `generate-groups.sh` look for vendor files if we have `-mod=vendor` here?

Reply:
Honestly: Dunno. I only copied this in from Serving :joy:.

Reply:
same in eventing: 
https://github.com/knative/eventing/blob/master/hack/update-codegen.sh#L22

Reply:
It looks like we don't actually invoke `generate-groups.sh` in `pkg`, possibly because we don't include any concrete CRDs here.